### PR TITLE
skip another example

### PR DESCRIPTION
**Pull request by nabobalis**
Should restore the doc build and has already committed to the other branches. 

Why this errors I can not tell you.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -42,6 +42,7 @@ jobs:
     envs:
       - macos: py36
         name: py36_test
+        posargs: -n=1
```
Comment (dstansby) [inline azure-pipelines.yml]: Why this change?

File: examples/example_template/example_template.py [hunk 2]
```diff
@@ -87,3 +80,11 @@ def dummy():
 # code directly will see the plots; this is not necessary for creating the docs
 
 plt.show()
+
```
Comment (dstansby) [inline examples/example_template/example_template.py]: And why has this block moved?

File: azure-pipelines.yml (hunk 1 above)
Comment (nabobalis) [reply azure-pipelines.yml]: Race condition on the macos builds only.

File: examples/example_template/example_template.py (hunk 2 above)
Comment (nabobalis) [reply examples/example_template/example_template.py]: To make sure it does not interrupt the flow of an example.

**Comment by nabobalis** [comment]
> Should probably also open a corresponding issue so we don't forget to fix the example.

I will open one.
